Solution.Versions.props

```
@@ -1,31 +1,31 @@
 <Project>
   <PropertyGroup>
     <!-- Solution dependencies -->
-    <AdysTechCredentialManagerVersion>1.7.0</AdysTechCredentialManagerVersion>
-    <AppInsightsWindowsDesktopVersion>2.10.42-preview</AppInsightsWindowsDesktopVersion>
-    <BenDemystifierVersion>0.1.4</BenDemystifierVersion>
-    <JetBrainsAnnotationsVersion>2018.2.1</JetBrainsAnnotationsVersion>
-    <LibGit2SharpVersion>0.25.0</LibGit2SharpVersion>
-    <MicrosoftVisualStudioCompositionVersion>15.6.36</MicrosoftVisualStudioCompositionVersion>
-    <MicrosoftVisualStudioThreadingVersion>16.5.132</MicrosoftVisualStudioThreadingVersion>
-    <NewtonsoftJsonVersion>10.0.3</NewtonsoftJsonVersion>
+    <AdysTechCredentialManagerVersion>2.1.1</AdysTechCredentialManagerVersion>
+    <AppInsightsWindowsDesktopVersion>2.13.1</AppInsightsWindowsDesktopVersion>
+    <BenDemystifierVersion>0.1.6</BenDemystifierVersion>
+    <JetBrainsAnnotationsVersion>2020.1.0 </JetBrainsAnnotationsVersion>
```

Just noticed stray spaces in few places throughout the file
```suggestion
    <JetBrainsAnnotationsVersion>2020.1.0</JetBrainsAnnotationsVersion>
```

---

👍